### PR TITLE
📝 Adding A `CONTRIBUTING.md` File

### PR DESCRIPTION
**Pull request by taljacob2**
A `CONTRIBUTING.md` file, in an open source
repository or site, provides potential project
contributors with a short guide to how they can
help with the project.

`CONTRIBUTING.md` is a file that contains all the
guidelines for contributing to JobSeeker
and its packages.

**Table Of Contents**
- Getting Started
- Conventions
- Issues & PRs
    - Issues
    - Pull Requests (PRs)

### You can view this file on my branch [here](https://github.com/taljacob2/JobSeeker/blob/doc/adding-contributing-md/CONTRIBUTING.md).

Signed-off-by: Tal Jacob <taljacob2@gmail.com>

### CODE REVIEW

**Review by paOmer — APPROVED** [review]
LGTM

**Review by Yarboa — COMMENTED** [review]
I would recommend adding:

- Contact through slack, with link
#os-07-group4 on [beyond slack](redhat-beyond.slack.com)

- Verify PR is ready for Core Review.
Please elaborate what actions needed

**Comment by taljacob2** [comment]
> I would recommend adding:
> 
> * Contact through slack, with link
>   #os-07-group4 on [beyond slack](redhat-beyond.slack.com)
> * Verify PR is ready for Core Review.
>   Please elaborate what actions needed

I have updated the file in [68ce24a](https://github.com/redhat-beyond/JobSeeker/commit/68ce24a88cbe40139960e8e55905f0bb68447f8a). You may review.

**Comment by Yarboa** [comment]
> > I would recommend adding:
> > 
> > * Contact through slack, with link
> >   #os-07-group4 on [beyond slack](redhat-beyond.slack.com)
> > * Verify PR is ready for Core Review.

This one should be discussed by all of you, In one of the following:
the PR itself, Issue related to PR or slack channel

> >   Please elaborate what actions needed
> 
> I have updated the file in [68ce24a](https://github.com/redhat-beyond/JobSeeker/commit/68ce24a88cbe40139960e8e55905f0bb68447f8a). You may review.

@paOmer @ShirleyFichman @DeanBiton @amittcohen 
see inline above

**Review by amittcohen — APPROVED** [review]
LGTM

**Review by DeanBiton — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Contributing To JobSeeker
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to JobSeeker and its packages, which are hosted in the [RedHat-Beyond](https://github.com/redhat-beyond) Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+You are welcome to contact us through slack, with link #os-07-group4 on [beyond slack](redhat-beyond.slack.com).
```
Comment (Yarboa) [inline CONTRIBUTING.md:7]: Consider adding contact header.
Please refer this contribution guide as sample
https://docs.openstack.org/neutron/latest/contributor/contributing.html

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Scan through our existing issues to find one that interests you. You can narrow down the search using labels as filters, or a specific word in the issue title. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+
+#### Commit Your Update
+
+Commit the changes once you are happy with them.
+
+Once your changes are ready, don't forget to self-review to speed up the review process :zap:.
+
+### Pull Requests (PRs)
```
Comment (Yarboa) [inline CONTRIBUTING.md:100]: Nothing about Test Actions? should pass successfully? 
how to check locally that your code is ready to be pushed?

Minimum of TeamMembers Approvals?
For ideas, 
you could look here
https://docs.openstack.org/neutron/latest/contributor/policies/code-reviews.html#code-review

**Review by Yarboa — CHANGES_REQUESTED** [review]
I would recommend adding:

- Contact through slack, with link
#os-07-group4 on [beyond slack](redhat-beyond.slack.com)

- Verify PR is ready for Core Review.
Please elaborate what actions needed

File: CONTRIBUTING.md (hunk 1 above)
Comment (taljacob2) [reply CONTRIBUTING.md:9]: I have updated the file in [6213999](https://github.com/redhat-beyond/JobSeeker/commit/621399924ef0c55d7971ff73ef460f7273254403). You may review.

File: CONTRIBUTING.md (hunk 2 above)
Comment (taljacob2) [reply CONTRIBUTING.md:122]: I have updated the file in [6213999](https://github.com/redhat-beyond/JobSeeker/commit/621399924ef0c55d7971ff73ef460f7273254403). You may review.

**Review by paOmer — APPROVED** [review]
LGTM!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+When you're finished with the changes, create a pull request, also known as a PR.
+
+- Verify the PR is ready for Core Review, and please elaborate what actions needed.
```
[the remark this answers is not in the document]
Comment (taljacob2) [reply CONTRIBUTING.md:126]: Will do.

**Review by Yarboa — COMMENTED** [review]
Most important thing is not here:

What are the steps PR is ready for CR review?
1) git hub actions PASSED ok
2) PRs should be small as much as possible
3) New features/Fixes must include tests
4) PR must be well explained (add link to the guideline in this page)
5) PR must be bound with Issue (add link to guideline in this page)
6) At least 2 contributes

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+### Our Team
+
+- [taljacob2](https://github.com/taljacob2)
+- [paOmer](https://github.com/paOmer)
+- [ShirleyFichman](https://github.com/ShirleyFichman)
+- [DeanBiton](https://github.com/DeanBiton)
+- [amittcohen](https://github.com/amittcohen)
+
+#### Our Mentors
```
[the remark this answers is not in the document]
Comment (taljacob2) [reply CONTRIBUTING.md]: Hi, you may review the update in [8a341d2](https://github.com/redhat-beyond/JobSeeker/commit/8a341d282f92576d2e1bcaf785d701bbf3d68cf3).

**Comment by taljacob2** [comment]
> Most important thing is not here:
> 
> What are the steps PR is ready for CR review?
> 
> 1. git hub actions PASSED ok
> 2. PRs should be small as much as possible
> 3. New features/Fixes must include tests
> 4. PR must be well explained (add link to the guideline in this page)
> 5. PR must be bound with Issue (add link to guideline in this page)
> 6. At least 2 contributes

I have added your notes in this commit: [8a341d2](https://github.com/redhat-beyond/JobSeeker/commit/8a341d282f92576d2e1bcaf785d701bbf3d68cf3). You may review.

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (taljacob2) [reply CONTRIBUTING.md]: I have added your notes in this commit: [8a341d2](https://github.com/redhat-beyond/JobSeeker/commit/8a341d282f92576d2e1bcaf785d701bbf3d68cf3). You may review.

**Review by Yarboa — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**